### PR TITLE
Add Leap Micro with combustion and make it default for PR testing

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?

Add Leap Micro to sumaform and use it by default for PR testing.

Because of Leap Micro being transactional, we need to install any additional package with combustion

### CODE REVIEW

**Review by srbarrios — APPROVED** [review]

File: backend_modules/libvirt/host/combustion [hunk 1]
```diff
@@ -27,6 +27,13 @@ zypper ar http://${ use_mirror_images ? mirror : "download.opensuse.org" }/repos
 zypper ar http://${ use_mirror_images ? mirror : "download.opensuse.org" }/update/leap/15.5/sle/ sle_update_repo
 %{ endif }
 
+for i in ${additional_repos};do
```
[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply backend_modules/libvirt/host/combustion:30]: This is bash, it would not work with the espaces

[the remark this answers is not in the document]
Comment (srbarrios) [reply backend_modules/libvirt/host/combustion:30]: Ah, I thouhg additional_repos was coming from Jinja2, not from bash.

File: salt/server_containerized/testsuite.sls [hunk 2]
```diff
@@ -110,11 +110,40 @@ repo_key_import:
       - cmd: galaxy_key_copy
 {% endif %}
 
+testing_overlay_devel_repo:
```
[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply salt/server_containerized/testsuite.sls:113]: I have no idea. I think this comes from your initial commit. Should I remove this overlay devel repo?

[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply salt/server_containerized/testsuite.sls]: fixed and rebased

**Review by cbosdo — APPROVED** [review]

**Pull request merged**